atom/browser/ui/views/win_frame_view.cc

```diff
@@ -40,12 +40,14 @@ int WinFrameView::NonClientHitTest(const gfx::Point& point) {
 }
 
 gfx::Size WinFrameView::GetMinimumSize() const {
-  gfx::Size size = FramelessView::GetMinimumSize();
+  gfx::Size size = window_->WindowSizeToFramelessSize(
+      window_->GetMinimumSize());
```

Comment:
I'm using ` window_->GetMinimumSize()` instead of `FramelessView::GetMinimumSize()`, to prevent potential issues, if we end up using `WindowSizeToFramelessSize` in `FramelessView` for OS X or Linux (the problem that comes to mind is that we'll remove the frame size twice).

This is true also for the `GetMaximumSize` code below.